main.go

```diff
@@ -229,6 +237,10 @@ func (p Ploop) Mount(target string, options map[string]string) (*flexvolume.Resp
 	if options["kubernetes.io/readwrite"] == "ro" {
 		readonly = true
 	}
+	if options["secretFromSystem"] == "true" {
+		options["kubernetes.io/secret/clusterName"] = os.Getenv("kubeVstorageClusterName")
+		options["kubernetes.io/secret/clusterPassword"] = os.Getenv("kubeVstorageClusterPassword")
+	}
```

Comment:
Is there any need to modify a function's argument ("options")? I prefer to treat them as constants and use local variables instead.